include/deal.II/base/bernstein_polynomial.h

```diff
@@ -0,0 +1,42 @@
+#include <deal.II/base/polynomial.h>
+#include <fstream>
+#include <iostream>
+
+#ifndef __deal2_bernstein_polynomial_h
+#define __deal2_bernstein_polynomial_h
+
+
+DEAL_II_NAMESPACE_OPEN
+
+/**
+ *  Class to compute the Bernstein basis polynomials of desire degree.
+ */
+template <typename number>
+class BernsteinPolynomial : public Polynomials::Polynomial<number>
```

Comment:
May I ask you to restructure your code to follow the naming convention we have in the library? This would require to rename the header, source and instantiation file to
```
polynomials_bernstein.h
polynomials_bernstein.cc
polynomials_bernstein.inst
```
and to rename the class itself to <code>PolynomialsBernstein</code>.